.install.sh

```diff
@@ -37,7 +37,8 @@ if [ "Windows_NT" != "$OS" ]; then
 else
   # Assumed to be Windows, default to x86
   if [[ "True" == "$IS_64BITS" ]]; then
-    cmake . -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON -G "Visual Studio 14 Win64"
+    # to build on win 10 with VS 16 use this command:
```

Comment:
Not sure this comment is still needed since this isn't just for Windows 10, but also other versions.

Reply:
While that's true, Win 7 just entered EOL so technically the only version of Windows we need to target is 10.

Reply:
There's still Windows 8.1 ;)

Reply:
Oh. Right. Does anybody actually use that, though? :)

Reply:
@isaacbrodsky, @dfellis : no strong opinion about it, just tell me what you (and the rest of the team) decide.

Reply:
I am not going to block the PR on that. If someone on the team feels strongly about it, they can change the comment when they do the patch bump to release it, I think.